### PR TITLE
fix identify and setting for user profile

### PR DESCRIPTION
**Pull request by icerove**
1. add function `identity` for user profile setting pre-requisite
2. fix some setting for user profile setting.

### CODE REVIEW

File: src/middleware/index.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ const readyStatePromise = store => next => action => {
 
 if (process.env.MIXPANEL_TOKEN) {
     mixpanel.init(process.env.MIXPANEL_TOKEN, {'property_blacklist': ['$current_url']})
+    window.mixpanel = mixpanel
```
Comment (vgrichina) [inline src/middleware/index.js:49]: is this debug leftover?

File: src/middleware/index.js [hunk 2]
```diff
@@ -69,31 +70,31 @@ const analyticsMiddleware = store => next => action => {
             timestamp: new Date().toString(),
             account_id: wallet.accountId
         })
+        let id = mixpanel.get_distinct_id();
+        mixpanel.identify(id);
```
Comment (vgrichina) [inline src/middleware/index.js:74]:
```
mixpanel.get_distinct_id

Returns the current distinct id of the user. This is either the id automatically generated by the library or the id that has been passed by a call to identify().
```

@icerove not sure I understand the idea here. Docs say that `.get_distinct_id` returns ID set by `.identify`

Can you explain big picture of what are you trying to achieve?

File: src/middleware/index.js [hunk 3]
```diff
@@ -69,31 +70,31 @@ const analyticsMiddleware = store => next => action => {
...
         mixpanel.people.set({
             network_id: networkId,
             stake: 0
         })
         if (trackingEvents.includes(action.type)){
             mixpanel.track(action.type, details);
         }
-        if (action.type === 'CREATE_NEW_ACCOUNT'){
```
Comment (vgrichina) [inline src/middleware/index.js:79]: Following changes don't seem to match PR topic of __fix identify__.

So either PR description should be updated or these changes should be in the separate PR with separate explanation.

File: src/middleware/index.js (hunk 2 above)
Comment (icerove) [reply src/middleware/index.js:74]: if I want to set up user profile, I have to call function `identify` and then use function `people.set`. see this: https://developer.mixpanel.com/docs/javascript#setting-profile-properties

File: src/middleware/index.js (hunk 1 above)
Comment (icerove) [reply src/middleware/index.js:49]: actually, not. I am thinking leaving this. But I will delete this for now since this is not for this PR.

File: src/middleware/index.js (hunk 3 above)
Comment (icerove) [reply src/middleware/index.js:79]: it is part of setting user profile and I will change the description.

File: src/middleware/index.js (hunk 1 above)
Comment (vgrichina) [reply src/middleware/index.js]: @icerove why is it necessary to keep `window.mixpanel`?

Comment (icerove) [reply src/middleware/index.js]: I used to think of easy checking for us.
